modules/apps/notification/notification-service/service.xml

```diff
@@ -171,7 +171,9 @@
 	</entity>
 	<exceptions>
 		<exception>NotificationTemplateAttachmentObjectFieldId</exception>
+		<exception>NotificationTemplateEditorType</exception>
 		<exception>NotificationTemplateFrom</exception>
+		<exception>NotificationTemplateFromName</exception>
```

Comment:
I think Brian wont accept this exception since we don't have a field called `FromName` inside `NotificationTemplate` entity anymore. Can you create a `NotificationRecipientSettingException` and use it?